robottelo/constants.py

```diff
@@ -48,10 +48,12 @@
 
 OPERATING_SYSTEMS = [
```

Comment:
I don't see any issue about having imports on constants modules, it is a python module and then can have any construct. Can you explain more about it, please? I may be missing something.

My suggestion was to avoid to maintain the list on two different places since it can be easily get out of sync.